### PR TITLE
ScanController Tests

### PR DESCRIPTION
**Pull request by drewb8L**
With this addition, the  establishment code scann funtionality it tested for valid and invalid inputs

### CODE REVIEW

File: Tests/ScanControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    _scanService.Setup(r => r.GetEstablishmentProducts(establishment.EstablishmentCode))
+      .ReturnsAsync(sr);
+    _controller = new ScanController(_scanService.Object);
+
+    var result = await _controller.GetEstablishment(establishment.EstablishmentCode);
+
+    var obj = result.Result as ObjectResult;
+    
+    Assert.Equal(200, obj.StatusCode);
+
```
Comment (digitopolis) [inline Tests/ScanControllerTests.cs]: With a test called `GetListOfProductsByEstablishmentCode` I would expect to see some assertions against a list of products being returned. Checking the `200` status code makes sense, but it's not fully testing the `GetEstablishment` controller action yet

File: Tests/ScanControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    {
+      Data = establishment.Products
+    };
+    _scanService.Setup(r => r.GetEstablishmentProducts(establishment.EstablishmentCode))
+      .ReturnsAsync(sr);
+    _controller = new ScanController(_scanService.Object);
+
+    var result = await _controller.GetEstablishment("Invalid Establishment Code");
+    Assert.Null( result.Value );
+
```
Comment (digitopolis) [inline Tests/ScanControllerTests.cs]: Kind of similarly here, why not do something like `Assert.Equal(404, obj.StatusCode)`?

File: Tests/ScanControllerTests.cs (hunk 1 above)
Comment (drewb8L) [reply Tests/ScanControllerTests.cs]: These are now refactored into 3 test, check for 200, 404 and list of products

File: Tests/ScanControllerTests.cs (hunk 2 above)
Comment (drewb8L) [reply Tests/ScanControllerTests.cs]: Changed to, `Assert.Contains("404", result.ToJson());`

**Pull request merged**